quodlibet/quodlibet/browsers/paned/prefs.py

```diff
@@ -18,13 +19,45 @@
 from quodlibet.qltk.x import SymbolicIconImage, MenuItem, Button
 from quodlibet.qltk import Icons
 from quodlibet.qltk.menubutton import MenuButton
-from quodlibet.qltk.ccb import ConfigCheckMenuItem, ConfigCheckButton
+from quodlibet.qltk.ccb import ConfigCheckButton
 from quodlibet.util import connect_obj, escape
+from quodlibet.const import COLUMN_MODE_SMALL, COLUMN_MODE_WIDE, \
+    COLUMN_MODE_COLUMNAR
 from .util import get_headers, save_headers
 
 
-class PatternEditor(Gtk.VBox):
+class ColumnModes(Gtk.VBox):
+    modes = [COLUMN_MODE_SMALL, COLUMN_MODE_WIDE, COLUMN_MODE_COLUMNAR]
+    buttons = []
```

Comment:
Move this into __init__ and rename to self.buttons, otherwise the buttons won't work the second time the pref window is opened